{
    "type": "bundle",
    "id": "bundle--ef220137-6882-42d0-8a61-6280c973e2d6",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--686817f9-904b-4372-989f-e331e0077699",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2021-03-13T20:12:50.552876Z",
            "modified": "2021-03-13T20:12:50.552876Z",
            "name": "CVE-2018-7251",
            "description": "An issue was discovered in config/error.php in Anchor 0.12.3. The error log is exposed at an errors.log URI, and contains MySQL credentials if a MySQL error (such as \"Too many connections\") has occurred.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2018-7251"
                }
            ],
            "object_marking_refs": [
                "marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"
            ]
        }
    ]
}